lldb/test/API/lang/swift/embedded/frame_variable/TestSwiftEmbeddedFrameVariable.py

```diff
@@ -129,3 +129,61 @@ def test(self):
                 "subSubField = (field = 4.2000000000000002)",
             ],
         )
+
+        self.expect(
```

Comment:
It would be more stable to use `lldbutil.check_variable` here.

Reply:
`check_variable` doesn't allow for checking for substrings, which I need in most of these tests

Reply:
What I meant was to write this as a chain of
```
var = frame.FindVariable("foo")
child = var.GetChildAtIndex(0)
lldbutil.check_var(child, ...)
```
So each check is precise and not depending on the formatting of the aggregate type.

Reply:
Ok, I'll do that in an NFC patch, as that file has a lot of checks and will take a while to convert them all